google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateInstanceRequest.java

```diff
@@ -146,6 +147,28 @@ public CreateInstanceRequest addCluster(
     return this;
   }
 
+  /**
+   * Adds a DEVELOPMENT cluster to the instance request.
+   *
+   * <p>This instance will have exactly one node cluster.
+   *
+   * @param clusterId The name of the cluster.
```

Comment:
per Oracle guidelines, "the name of the cluster" (no initial cap, no period unless it's a complete sentence)

Reply:
Thanks for the details, Again have updated the param description accordingly.